CMakeLists.txt

```diff
@@ -554,6 +554,13 @@ install(
   PATTERN "CMakeLists.txt" EXCLUDE
 )
 
+# To override this, use '-DWL_INSTALL_BINDIR=<absolute path>' when configuring or change
+# 'CMAKE_INSTALL_PREFIX'.
+# Example: to have the bin installed to "/usr/games", just use '-DCMAKE_INSTALL_PREFIX=/usr'
+# and don't specify '-DWL_INSTALL_BINDIR' when configuring.
+# Note: 'CMAKE_INSTALL_PREFIX' defaults to /usr/local on UNIX and c:/Program Files/${PROJECT_NAME} on Windows
+set(WL_INSTALL_BINDIR "${CMAKE_INSTALL_PREFIX}/games" CACHE PATH "Destination of executable files (if installing)")
```

Comment:
Does anyone think we need a condition  here to specify a different default directory if installing on Windows? If I read the cmake docs right, the binaries would get installed to `c:/Program Files/widelands/games/`

Reply:
Windows builds do not get installed by `make` but instead use the innosetup, mac os also has a separate install method. This should be unaffected by this change. However to be sure I will test the Windows installer and maybe @klaus-halfmann can check on MacOS (go to "Checks" -> "Artifacts")